### DISCUSSION TITLE
Github pages published but err_connection_timed_out

### DISCUSSION THREAD

**Discussion by zaykha** [community/community#22365] [Other Feature Feedback, Questions, & Ideas]
<p>Hello,<br>
I have uploaded my first practice static website with 1 html, 2 htms , 3js ,3css and an image folder.<br>
My proxy and connections are fine. I tried putting 2 htm files in a separate folder also. I also tried waiting a day for the website to load. Tried using my mobile data to load the page too.<br>
But its the same err connection timed out.<br>
Please help me solve this issue. thank you Github and all others in advance.</p><aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/zaykha/HappyNest">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/zaykha/HappyNest" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/79003614/181070563-64beaedb-994f-407e-ac07-874a511a00cb.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/6/f/6f6754d5cc56a3c9a0970d111854e7b6f7199d70_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/6/f/6f6754d5cc56a3c9a0970d111854e7b6f7199d70_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/6/f/6f6754d5cc56a3c9a0970d111854e7b6f7199d70.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/6/f/6f6754d5cc56a3c9a0970d111854e7b6f7199d70_2_10x10.png"></div>

<h3><a href="https://github.com/zaykha/HappyNest" target="_blank" rel="noopener nofollow ugc">GitHub - zaykha/HappyNest: First unguided Project</a></h3>

  <p>First unguided Project. Contribute to zaykha/HappyNest development by creating an account on GitHub.</p>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>
<p>
link to my public repo.</p>

**Comment by tcbyrd** [comment, marked as answer]
<p>Hey there <a class="mention" href="/zaykha">@zaykha</a>! Thanks for trying out GitHub Pages. I’m not sure about the “connection timed out” part. The site works for me, but is published at the URL shown in the build result here: <a href="https://github.com/zaykha/HappyNest/runs/4432656757" class="inline-onebox">Set theme jekyll-theme-slate · zaykha/HappyNest@92ccd90 · GitHub</a></p>
<p>Keep in mind that <code>happynest1.github.io</code> cannot be used here on your user account. The <code>happynest1</code> part of that domain maps to a user/organization on GitHub, which doesn’t exist. Your Pages site is at <code>zaykha.github.io</code>. You can read about how URLs map to pages on this page:</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#types-of-github-pages-sites">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">

      <a href="https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#types-of-github-pages-sites" target="_blank" rel="noopener">GitHub Docs</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/13207348/181070564-1d8b8843-1eb5-4d55-86ca-05feeb4cf409.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png">

<h3><a href="https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#types-of-github-pages-sites" target="_blank" rel="noopener">About GitHub Pages - GitHub Docs</a></h3>



  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>If you’d like to setup a custom URL instead of one that includes your username, look into setting up a custom domain:</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">

      <a href="https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site" target="_blank" rel="noopener">GitHub Docs</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/13207348/181070565-37a01a85-2184-43fe-98d2-8cb006fc8e5b.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png">

<h3><a href="https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site" target="_blank" rel="noopener">Configuring a custom domain for your GitHub Pages site - GitHub Docs</a></h3>



  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>Hope that helps!</p>

**Comment by bencat-sixense** [comment]
<p>I experience many issues with a few Github services since 1 hour, maybe you’ve the same…</p>

**Comment by zaykha** [comment]
<p>thank you sir. I found that my ISP has block almost all website coz its in Myanmar, so i used VPN and its showing. but now new issue is that images are not shown, may be fail to retrieve images from ‘images’ files.<br>
do i have to upload all images without folder and change directories in all my htmls and htms files? or any other solution.<br>
currently "  background: url(/images/promo1dec.jpg) no-repeat center;" in my css file.<br>
thanks for your help again sir.</p>

**Comment by tcbyrd** [comment]
<aside class="quote no-group" data-username="zaykha" data-post="4" data-topic="216020">
<div class="title">
<div class="quote-controls"></div>
 zaykha:</div>
<blockquote>
<p>currently " background: url(/images/promo1dec.jpg) no-repeat center;" in my css file.</p>
</blockquote>
</aside>
<p>Since you’re using a project page, you’ll want to include the name of the repo in the path name. For example:</p>
<pre><code class="lang-auto">background: url('/happynest/images/promo1dec.jpg')
</code></pre>
<p>You can see your image loads at <a href="https://zaykha.github.io/HappyNest/images/promo1dec.jpg">https://zaykha.github.io/HappyNest/images/promo1dec.jpg</a></p>
